packages/orbit-components/src/Separator/Separator.stories.jsx

```diff
@@ -25,7 +26,10 @@ export const Playground = (): React.Node => {
     [null, ...Object.values(SPACINGS_AFTER)],
     SPACINGS_AFTER.LARGEST,
   );
-  return <Separator spaceAfter={spaceAfter} />;
+  const align = select("align", ["left", "right", "center"], "left");
```

Comment:
could you also add that to docs? 🙏🏻

Reply:
What do you mean? I added `align` prop to `Separator/README.md` already.

Reply:
I meant to add them to our sandbox